scripts/configure.js

```diff
@@ -323,6 +323,10 @@ const getConfig = (() => {
             "metro.config.js": {
               source: path.join(testAppPath, "example", "metro.config.js"),
             },
+            ".gitignore": fs
+              .readFileSync(path.join(process.cwd(), "example", ".gitignore"))
```

Comment:
You shouldn't use current directory. We don't know where the user might be calling this script from. You should use `testAppPath` instead.